### PR TITLE
Add Salesforce to powered-by page

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
ping @miguno for review as well

**Comment by bbejeck** [comment]
Validated rendering and links by running Apache server locally.  The entry renders like this:

Screen Shot 2021-08-24 at 11 38 43 AM

File: powered-by.html [hunk 1]
```diff
@@ -499,6 +499,11 @@
         "logo": "rollbar.png",
         "logoBgColor": "#ffffff",
         "description": "Rollbar uses Kafka at large scale to store all incoming raw blobs. Kafka allowed us to have multiple workers and also allowed us to migrate to Kubernetes."
+    }, {
+        "link": "https://www.salesforce.com/",
+        "logo": "salesforce.jpg",
+        "logoBgColor": "#ffffff",
+        "description" :  "Salesforce adopted Apache Kafka to implement a <a href='https://engineering.salesforce.com/expanding-visibility-with-apache-kafka-e305b12c4aba' , target='_blank'> pub/sub architecture system </a> and to securely add an enterprise-ready, <a href='https://engineering.salesforce.com/how-apache-kafka-inspired-our-platform-events-architecture-2f351fe4cf63', target='_blank'>event-driven layer</a> to our multitenant system. With Kafka as the central nervous system of our microservice architecture, <a href='https://engineering.salesforce.com/real-time-einstein-insights-using-kafka-streams-ca94008c2c6f', target='_blank'> Kafka Streams jobs </a> perform a variety of operations to generate useful real-time insights for our customers."
```
Comment (miguno) [inline powered-by.html]:
```suggestion
        "description" :  "Salesforce adopted Apache Kafka to implement a <a href='https://engineering.salesforce.com/expanding-visibility-with-apache-kafka-e305b12c4aba' , target='_blank'>pub/sub architecture system</a> and to securely add an enterprise-ready, <a href='https://engineering.salesforce.com/how-apache-kafka-inspired-our-platform-events-architecture-2f351fe4cf63', target='_blank'>event-driven layer</a> to our multi-tenant system. With Kafka as the central nervous system of our microservices architecture, <a href='https://engineering.salesforce.com/real-time-einstein-insights-using-kafka-streams-ca94008c2c6f', target='_blank'>Kafka Streams jobs</a> perform a variety of operations to generate useful real-time insights for our customers."
```

Comment (miguno) [reply powered-by.html]: Can we rename "Kafka Streams jobs" to "Kafka Streams applications"?

**Comment by rohitmdesh** [comment]
👍

**Review by cadonna — APPROVED** [review]
Thanks, @bbejeck !

LGTM!

**Comment by bbejeck** [comment]
Thanks for the review @cadonna!

**Pull request merged**